### PR TITLE
Fix new address book submit button border color

### LINKED ISSUE #3067: Minor layout issue: Submit button for "New address book" is red
### Steps to reproduce

1. Go to Contacts app
2. Click on Contacts settings

The "submit" button for adding address books is outlined red, which looks weird. Probably a minor CSS or state issue.

See screenshot:

Bildschirmfoto 2022-10-20 um 19 49 05

### Expected behavior

The "submit" button should not be outlined in red

### Actual behavior

The "submit" button should is outlined in red

### Server operating system

Debian 11

### Web server

Apache

### Database engine version

MariaDB

### PHP engine version

PHP 8.1

### Nextcloud version

25.0.0

### Updated from an older installed version or fresh install

Updated from 24

### Signing status

```Text
Technical information
=====================
The following list covers which files have failed the integrity check. Please read
the previous linked documentation to learn more about the errors and how to fix
them.

Results
=======
- related_resources
	- EXCEPTION
		- OC\IntegrityCheck\Exceptions\InvalidSignatureException
		- Signature data not found.

Raw output
==========
Array
(
    [related_resources] => Array
        (
            [EXCEPTION] => Array
                (
                    [class] => OC\IntegrityCheck\Exceptions\InvalidSignatureException
                    [message] => Signature data not found.
                )

        )

)
```

### Browser

Chrome

### Client operating system

macOS

### PR DESCRIPTION
**Pull request by aryatalathi**
Fixes #3067

### CODE REVIEW

**Comment by ChristophWurst** [comment]
Thanks a lot for the contribution @aryatalathi! For design changes it's recommended to post a before and after screenshot that shows the UI before your change and after. Could you add this here as a comment? Thanks!

**Comment by ChristophWurst** [comment]
Hey @aryatalathi,

Could you please post a *before* and *after* screenshot to demo your changes?

Cheers

**Comment by ChristophWurst** [comment]
Hey @aryatalathi,

Could you please post a before and after screenshot to demo your changes?

Cheers

File: src/components/AppNavigation/ContactsSettings.vue [hunk 1]
```diff
@@ -170,7 +170,7 @@ export default {
 .settings-new-addressbook >>> .icon-confirm {
 	min-height: 44px;
 	height: 44px;
-	border-color: var(--color-border-dark) !important;
+	border-color: black !important;
```
Comment (ChristophWurst) [inline src/components/AppNavigation/ContactsSettings.vue:173]: I'm convinced this should use a css var, not a hard-coded value. The black border will be problematic with a bright theme.

**Review by ChristophWurst — CHANGES_REQUESTED** [review]

**Comment by SebastianKrupinski** [comment]
This issue is no longer applicable.

**Pull request closed without merging**